help/user-guide/isajson-templates.md

```diff
@@ -36,4 +46,19 @@ Project members have the capability to create Experiment Sample Templates for th
 * If you want to add new attributes of your choice to your new template, select “Add new attribute” button.
 * Fill out the mandatory and optional fields. Note that for ISA-JSON compliant Experiments, the ISA Tag is a mandatory field.
 * For ISA tag, note that the following tags can only be present once in a template: source, sample, other_material, data_file, protocol. One of these tags is already present in existing templates, so select one of the following as ISA tags: source_characteristic, parameter_value, sample_characteristic, other_material_characteristic, data_file_comment.
-6. Select "Create".
\ No newline at end of file
+6. Select "Create".
+
+
+### JSON upload via Server admin panel [FOR SYSTEM ADMINISTRATORS]
```

Comment:
please, keep the distinction "for instance administrator" and "for project members" as main title (1 #).